pom.xml

```diff
@@ -290,7 +290,7 @@
         <dependency>
             <groupId>org.apache.kafka</groupId>
             <artifactId>connect-api</artifactId>
-            <version>3.2.0</version>
```

Comment:
Lets make a change in our documentation. we still mention we use 2.8.1 (unfortunately)
CC: @sfc-gh-lema https://docs.snowflake.com/en/user-guide/kafka-connector-install.html#installation-prerequisites (Search for 2.8.1)

We should also fix what all AK and confluent versions we support for brokers. we have tests against 5.5.11, 6.2.x and 7.2.x. Our documentation is very old and outdated.

Reply:
@sfc-gh-japatel thanks for letting me know. I'll create a JIRA and update the documentation.